cfg/rippled-example.cfg

```diff
@@ -1617,7 +1617,7 @@ admin = 127.0.0.1
 protocol = http
 
 [port_peer]
-port = 51235
+port = 2459
```

Comment:
I don't think you want to change this.

Reply:
Apparently, 51235 conflicts with dhcp on Windows. There isn't really a special significance to using 51235 for the peer port. 2459 is the IANA registered port and the [default](https://github.com/XRPLF/rippled/blob/0ce15e0e358996b5371d2adc83e911cc7a3a7225/src/ripple/protocol/SystemParameters.h#L80-L81) used when connecting to other peers (when a port is not specified).

Per https://github.com/XRPLF/rippled/issues/3037, it is probably time to change the port to 2459 in this example config. Operators can always set it to 51235 or whatever they want; this file is just an example.

Reply:
Oh I see this is officially the port for xrpl. 
I guess it should have been changed 3 years ago.